test/clj/rems/browser_test_util.clj

```diff
@@ -543,7 +543,7 @@
   Returns the test report.
 
   Ignores:
-  - all divs of body except #app and a re-frisk unnamed div
+  - all divs of body except #app
```

Comment:
Here only the comment is removed but the selector is not touched? Perhaps the `'body > div:not(#app)'` could be left out? Well maybe we do want to keep it since all our stuff is inside `#app`.

Reply:
let's keep the selector for now.